docs/application/vstools/tools/manifest-editor.md

```diff
@@ -26,14 +28,13 @@ There are 2 different ways to edit the `tizen-manifest.xml` file:
 > The `tizen-manifest.xml` file must conform to both the standard XML file format and the Tizen .NET application specification requirements. Editing the manifest file XML structure with the text editor is intended for advanced users only. If the file does not conform to the standard and the requirements, errors can occur during installation.
 
 
-
-
+<a name="hierarchy"></a>
 ## Manifest Element Hierarchy
 
 The Tizen .NET application manifest file consists of XML elements organized in a hierarchy. The following tree structure shows the relationship between the elements of the `tizen-manifest.xml` file.
 
 | `<manifest>` |                                   |                          |                  |
```

Comment:
It looks like you removed hyperlinks in this chart on purpose, didn't you?

Reply:
The commit is based on the bran 'convert-md'  in tizen public repo.
=============================================
commit 744c8cbc0565bbb9f6e5197c6c0b08a09a74d8ab
Author: Young-Ae Kang <youngae.kang@samsung.com>
Date:   Tue Jan 2 17:29:10 2018 +0900

    Update the latest VS Tools for Tizen on vstools
    
     - diff old and new.
     - old commit: a6ed34a74d32b200e2668448d203092463c61895
     - new commit: 39b7663eb339dcb7b608a78b105e2fce5c30cad1 <== **This commit is the latest commit that we converted HTML to MD**.
    
    Change-Id: Ic73fac4c8826cae8fd1c5c137495c297bac1be5d
=============================================
The commit was committed on 2th Jan.  The latest commit including vstools is 395b21c537d36f1666aac7b42e9f987f909c0de0 merged on 4th Jan.

We didn't convert the latest commit of public git. You steal have more commits that must be converted. We are supposed to update some files sooner or later.

Moreover, I think we don't need to provide hyperlink to each element because the new docs site will create TOC of ##, ###, and maybe #### tage. You can access each element on the right site.

The manual updates may make more mistakes.